clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonLoadBalancerProvider.groovy

```diff
@@ -63,15 +63,19 @@ class AmazonLoadBalancerProvider implements LoadBalancerProvider<AmazonLoadBalan
     CacheData application = cacheView.get(APPLICATIONS.ns, Keys.getApplicationKey(applicationName))
 
     def applicationServerGroups = application ? resolveRelationshipData(application, SERVER_GROUPS.ns) : []
+    def allLoadBalancerKeys = cacheView.getIdentifiers(LOAD_BALANCERS.ns)
```

Comment:
might still be worth doing `cacheView.filterIdentifiers(LOAD_BALANCERS.ns, Keys.getLoadBalancerKey(applicationName + '*', '*', '*', null)` which will map to a glob of `aws:loadBalancers:*:*:applicationName*`